apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/extract/s3/S3RifFile.java

```diff
@@ -60,13 +60,13 @@ public S3RifFile(
     this.manifestEntryDownload = manifestEntryDownload;
   }
 
-  /** @see gov.cms.bfd.model.rif.RifFile#getFileType() */
+  /** Gets the file type of the static rif resource as a {@link RifFileType}. */
```

Comment:
Missed the inerhitdocs in this class. Last thing to fix.

Reply:
done